### PR TITLE
[SwiftLanguageRuntime] At this point, we have a fully realized type.

### PR DESCRIPTION
**Pull request by dcci**

### CODE REVIEW

**Comment by dcci** [comment]
@swift-ci test and merge

File: source/Target/SwiftLanguageRuntime.cpp [hunk 1]
```diff
@@ -2595,16 +2595,8 @@ bool SwiftLanguageRuntime::CouldHaveDynamicValue(ValueObject &in_value) {
...
-    // that is
-    // a good job for dynamic types to perform
-    if (in_value.IsBaseClass()) {
-      CompilerType base_type(in_value.GetCompilerType());
-      if (SwiftASTContext::IsFullyRealized(base_type))
-        return false;
-    }
-    return true;
+    // disable it.
+    return !in_value.IsBaseClass();
```
Comment (adrian-prantl) [inline source/Target/SwiftLanguageRuntime.cpp:2599]: Do you want to make this an assertion instead?

**Comment by dcci** [comment]
```
14:27:11 ********************
14:27:11 Unresolved Tests (1):
14:27:11     lldb-Suite :: lang/swift/implementation_only_imports/main_executable/TestMainExecutable.py
14:27:11 
```

@jrose-apple @adrian-prantl this test seems to fail on&off.

**Comment by dcci** [comment]
https://ci.swift.org/job/swift-lldb-PR-osx/2211/

**Comment by dcci** [comment]
@swift-ci test macos platform

**Comment by dcci** [comment]
@swift-ci test macos platform

**Comment by dcci** [comment]
@swift-ci test macos platform

**Pull request merged**